App builder setup info

jhadobe opened the pull request:
## Purpose of this pull request

This pull requests updates `requirements.md` with links to resources for Commerce and App Builder

jeff-matthews commented on src/pages/guides/sellers/compatibility/requirements.md:
```diff
@@ -14,3 +14,18 @@ To mitigate this, Adobe has introduced multiple requirements for listings that a
 
 The topics in this section describe these requirements in more detail. They're also covered in the Policies section of the Developer Portal (to see Policies, go to Resources in the top menu).
+
+## Commerce Requirements:
```
```suggestion
## Commerce requirements
```

jeff-matthews commented on src/pages/guides/sellers/compatibility/requirements.md:
```diff
@@ -14,3 +14,18 @@ To mitigate this, Adobe has introduced multiple requirements for listings that a
+- [Configuring Events for Adobe Commerce](https://developer.adobe.com/commerce/events/get-started/configure-commerce/)
+- [Setting up Admin UI SDK](https://developer.adobe.com/commerce/extensibility/admin-ui-sdk/)
+
+## App Builder Requirements:
```
```suggestion
## App Builder requirements
```

jeff-matthews reviewed (commented):
I think we need a lead-in paragraph that makes it clear that these requirements apply to apps specifically. That could mean removing the headings. Or, adding a parent heading.

jhadobe commented:
> I think we need a lead-in paragraph that makes it clear that these requirements apply to apps specifically. That could mean removing the headings. Or, adding a parent heading.

Thanks, @jeff-matthews. I have addressed your comments in the latest commits.

jeff-matthews reviewed (approved):
Looks good to me. 

@naomirobichaud, please confirm.
